KesaApplication/LoginViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewController.swift
+//  KesaApplication
+//
+//  Created by Jiwoo Lim on 2016. 6. 1..
+//  Copyright © 2016년 kesa. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    //Function to display error message
+    func displayAlertMessage(userMessage: String){
+        let myAlert = UIAlertController(title: "Error", message: userMessage, preferredStyle: UIAlertControllerStyle.Alert);
+        
+        let okAction = UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler:nil);
+        
+        myAlert.addAction(okAction);
+        
+        self.presentViewController(myAlert, animated: true, completion: nil);
+    }
+
+    //Two text fields
+    @IBOutlet weak var userEmailTextField: UITextField!
+    @IBOutlet weak var userPasswordTextField: UITextField!
+    
+    @IBAction func loginButtonTapped(sender: AnyObject) {
+        
+        let userEmail = userEmailTextField.text;
+        let userPassword = userPasswordTextField.text;
+        
+        let userEmailStored = NSUserDefaults.standardUserDefaults().stringForKey("userEmail");
+        let userPasswordStored = NSUserDefaults.standardUserDefaults().stringForKey("userPassword");
```

Comment:
These part of implementation should be replaced with methods in AccountManager class. Please use **authenticateWithPassword** method from **AccountManager** class to authenticate.

Make accountManager constant variable at the top of class
```
let accountManager:AccountManager = AccountManager()